components/sidebar/sidebar.jsx

```diff
@@ -648,21 +654,40 @@ export default class Sidebar extends React.PureComponent {
             );
         }
 
-        let quickSwitchTextShortcutId = 'quick_switch_modal.channelsShortcut.windows';
-        let quickSwitchTextShortcutDefault = '- CTRL+K';
-        if (Utils.isMac()) {
-            quickSwitchTextShortcutId = 'quick_switch_modal.channelsShortcut.mac';
-            quickSwitchTextShortcutDefault = '- ⌘K';
-        }
+        let quickSwitchText = null;
+        if (channelSwitcherOption) {
+            let quickSwitchTextShortcutId = 'quick_switch_modal.channelsShortcut.windows';
+            let quickSwitchTextShortcutDefault = '- CTRL+K';
+            if (Utils.isMac()) {
+                quickSwitchTextShortcutId = 'quick_switch_modal.channelsShortcut.mac';
```

Comment:
When you rebase from master, be sure this string is wrapped inside the `t` function from `utils/i18n`.

Reply:
done